storybook/src/admin/Tooltip.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { Tooltip } from "@comet/admin";
+import { Add } from "@comet/admin-icons";
+import { Box, Button, Card, CardContent, Stack, Typography } from "@mui/material";
+import { forwardRef, useState } from "react";
+
+export default {
+    title: "@comet/admin/Tooltip",
+};
+
+export const IconWithTooltip = {
+    render: () => {
+        return (
+            <Tooltip title="Add something">
+                <Add />
+            </Tooltip>
+        );
+    },
+
+    name: "Icon with Tooltip",
+};
+
+export const AllTooltipTriggers = {
+    render: function Render() {
+        const [clicks, setClicks] = useState(0);
+
+        const NonClickable = forwardRef<HTMLDivElement, { children: React.ReactNode }>(({ children }, ref) => {
```

Comment:
Why do you need the ref forwarding here?

Reply:
Ah sorry, tried something out and forgot to remove that 😅